components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -377,19 +377,37 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
         this.setState({ workspaceInstance, error });
     }
 
-    async ensureWorkspaceAuth(instanceID: string) {
+    async ensureWorkspaceAuth(instanceID: string, retry: boolean) {
         if (!document.cookie.includes(`${instanceID}_owner_`)) {
```

Comment:
Is there a constant/function we could re-use with the logic which sets that cookie? This would make it easier to ensure it remains consistent between the two places.

Reply:
Looks like this is implemented on the go-side in /workspace/gitpod/components/ws-proxy/pkg/proxy/auth.go so won't be able to share a constant.